lib/SensorFusion/src/App.cpp

```diff
@@ -106,7 +80,8 @@ static const uint32_t JSON_DOC_DEFAULT_SIZE = 1024U;
 void App::setup()
 {
     Serial.begin(SERIAL_BAUDRATE);
-
+    SensorFusion::getInstance().init();
```

Comment:
No error handling necessary?

Reply:
In the init-Function, only member variables without any external dependency are being initialized.